equinix/data_source_network_device.go

```diff
@@ -788,10 +798,14 @@ func updateDataSourceNetworkDeviceResource(primary *ne.Device, secondary *ne.Dev
 	if err := d.Set(neDeviceSchemaNames["ZoneCode"], primary.ZoneCode); err != nil {
 		return fmt.Errorf("error reading ZoneCode: %s", err)
 	}
+	if err := d.Set(neDeviceSchemaNames["ProjectID"], primary.ProjectID); err != nil {
+		return fmt.Errorf("error reading ProjectID: %s", err)
+	}
 	if secondary != nil {
 		if v, ok := d.GetOk(neDeviceSchemaNames["Secondary"]); ok {
 			secondaryFromSchema := expandNetworkDeviceSecondary(v.([]interface{}))
 			secondary.LicenseFile = secondaryFromSchema.LicenseFile
+			secondary.ProjectID = secondaryFromSchema.ProjectID
```

Comment:
As discussed, since project id is returned in GET device by uuid API, we don't need to populate it from schema itself.

Reply:
as discussed, we need this project_id to be in device details API response in secondary.

Reply:
The project id is included in the API, we don't need to read it from schema again.

Reply:
done